web/src/classic/components/organisms/Published/core/hooks.ts

```diff
@@ -203,6 +212,27 @@ export default (alias?: string) => {
     [],
   );
 
+  const selectedLayerId = useMemo(() => {
+    return selected?.type === "layer"
+      ? { layerId: selected.layerId, featureId: selected.featureId }
+      : undefined;
+  }, [selected]);
+
+  const layerSelectionReason = useMemo(
+    () => (selected?.type === "layer" ? selected.layerSelectionReason : undefined),
+    [selected],
+  );
```

Comment:
Really really small thing, but can you keep to one style. In this case, for selectedLayerId don't need return {}

Reply:
Updated. Thank you!